content/intro-to-storybook/svelte/pt/get-started.md

```diff
@@ -0,0 +1,76 @@
+---
+title: 'Tutorial Storybook para o Svelte'
+tocTitle: 'Get started'
```

Comment:
@yamgomes, if you're okay with this, can you also update this and try and matching the other Portuguese translations